nemo_skills/inference/eval/hilbench_assets/ask_human_server.py

```
@@ -0,0 +1,835 @@
+                        single_registry = registry_candidates[0]
+                if single_registry.exists():
+                    try:
+                        registry = load_blocker_registry(single_registry)
+                        aliases = _instance_id_aliases_for_task_dir(TASKS_DIR)
+                        for instance_id in aliases:
+                            CACHED_BLOCKERS[instance_id] = registry
+                        logger.info(
+                            f"Loaded {len(registry)} blocker(s) for instance aliases "
+                            f"{aliases} from {single_registry}"
+                        )
+                    except Exception as e:
```

Ruff (0.15.20)

[warning] Do not catch blind exception: `Exception`

(BLE001)